From 0c8d2d78065f8358440d2d5a5dc7704f04617935 Mon Sep 17 00:00:00 2001
From: lyz <lyz@rock-chips.com>
Date: Fri, 19 Sep 2014 08:48:27 +0800
Subject: [PATCH] HSIC: fix compilation error

usbdev_rk32.c: In function 'rk_hsic_soft_reset':
usbdev_rk32.c:438: error: storage size of 'rst_hsic_a' isn't known
usbdev_rk32.c:438: error: storage size of 'rst_hsic_p' isn't known
---
 drivers/usb/dwc_otg_310/usbdev_rk32.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/dwc_otg_310/usbdev_rk32.c b/drivers/usb/dwc_otg_310/usbdev_rk32.c
index 5a59c0bb811e..6d28789d758d 100755
--- a/drivers/usb/dwc_otg_310/usbdev_rk32.c
+++ b/drivers/usb/dwc_otg_310/usbdev_rk32.c
@@ -435,7 +435,7 @@ static void rk_hsic_clock_enable(void *pdata, int enable)
 static void rk_hsic_soft_reset(void *pdata, enum rkusb_rst_flag rst_type)
 {
 	struct rkehci_platform_data *usbpdata = pdata;
-	struct reset_control *rst_hsic_h, rst_hsic_a, rst_hsic_p;
+	struct reset_control *rst_hsic_h, *rst_hsic_a, *rst_hsic_p;
 
 	rst_hsic_h = devm_reset_control_get(usbpdata->dev, "hsic_ahb");
 	rst_hsic_a = devm_reset_control_get(usbpdata->dev, "hsic_aux");
-- 
2.35.3

